kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -125,8 +125,7 @@ def kevm_prove(
                 cut_point_rules=cut_point_rules,
             )
             assert isinstance(proof, APRProof)
-            failure_nodes = proof.failing
-            if len(failure_nodes) == 0:
+            if proof.status == ProofStatus.PASSED:
```

Comment:
```suggestion
            if proof.passed:
```

We can use this helper, defined here: https://github.com/runtimeverification/pyk/blob/949163ea8cc7f168490c84a7daa5df5481d1af9c/src/pyk/proof/proof.py#L182

Reply:
It looks like there are other places in this file where `ProofStatus.PASSED` or `ProofStatus.FAILED` is compared to directly instead of using `Proof.passed` and `Proof.failed`, can you adjust those as well?

Reply:
@ehildenb Done, please check if you can - I've also removed some dead code.